README.md

```diff
@@ -250,7 +253,7 @@ Support tickets for RedHat Trusted Profile Analyzer can be opened at https://acc
 
 ## Release notes and Roadmap
 
-Release notes can be found [here](https://docs.redhat.com/en/documentation/red_hat_trusted_profile_analyzer/1.2/html/release_notes/index).
+Release notes can be found [here](https://access.redhat.com/products/red-hat-trusted-profile-analyzer).
```

Comment:
Please revert this change.  The URL that was previously there was correct.

Reply:
ok